krystal-visualization/src/main/java/com/flipkart/krystal/visualization/models/Graph.java

```diff
@@ -0,0 +1,12 @@
+package com.flipkart.krystal.visualization.models;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
Change all @Data to @Value to make these classes immutable unless you really need mutability